modules/RTC/TPCUtils.js

```diff
@@ -253,17 +253,11 @@ export class TPCUtils {
         }
 
         logger.debug(`Removing ${localTrack} on ${this.pc}`);
-        transceiver.sender.replaceTrack(null)
+
+        return transceiver.sender.replaceTrack(null)
```

Comment:
why the catch and the return value is being removed? the JSDoc returns signature is not updated

Reply:
have you checked how is this result propagated further up the chain?